### ISSUE TITLE
MAGs percent of total assembled contigs

### ISSUE THREAD

**Issue by TJrogers86** [bxlab/metaWRAP#302]
I was wondering if there is a stats file that tells you the percent of total assembled configs that went into the final MAGs? If not, do you know how I might find that number?

**Comment by ursky** [comment]
No, but that should be trivial enough to calculate. Get the number of binned contigs: `cat bins_folder/*fa | grep ">" | wc -l`. Get the total number of contigs: `grep ">" full_assembly.fa | wc -l`. And then divide the numbers. Of course, that doesn't tell the full story because most of the unbinned contigs will be very short (<3kb), so it might look worse than it is. If youre curious of the total sequence length binned instead you could just compare the number of nucleotides in the bins `cat bins_folder/*fa | grep -v ">" | wc -c` with the number of nucleotides in the assembly `grep -v ">" full_assembly.fa | wc -c`.

**Comment by TJrogers86** [comment]
Thanks! and yes, the length calculation is preferable. I honestly didnt know I could do that with grep.
